### PR TITLE
Fix/add product after migration

### PR DESCRIPTION
**Pull request by Tkachenko01001**

### CODE REVIEW

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx [hunk 1]
```diff
@@ -12,31 +12,36 @@ import {
...
-      if (productId) {
-        productData.price.forEach((price) => {
-          dispatch(addPrice({ price, productId: productId! }))
-            .unwrap()
-            .catch((error) => {
-              return console.log(error)
-            })
-        })
-      }
+  const sendPricesSequentially = async (productId) => {
```
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx:30]: you forgot type for `productId`

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx [hunk 2]
```diff
@@ -12,31 +12,36 @@ import {
...
+} from '../../redux/crm-add-new-product/slice/product'
 import { ProductStatus } from '../../types'
 
 const CrmAddNewProductButton = () => {
-  const productData = useSelector((state: RootState) => state.newProduct)
+  const productData = useSelector((state: ProductState) => state.newProduct)
+
   const productId = useSelector(
-    (state: RootState) => state.newProduct.productId
+    (state: ProductState) => state.newProduct.productId
```
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx:26]: why `ProductState` and not `RootState`?

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx [hunk 3]
```diff
@@ -12,31 +12,36 @@ import {
...
-            })
-        })
-      }
+  const sendPricesSequentially = async (productId) => {
+    if (productId) {
+      productData.price.forEach((price) => {
+        dispatch(addPrice({ price, productId: productId! }))
+          .unwrap()
+          .catch((error) => {
+            return console.log(error)
```
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx:36]:
```suggestion
           console.log(error)
```

File: src/components/Crm-images/index.tsx [hunk 4]
```diff
@@ -26,8 +26,12 @@ const CrmImages = () => {
   const [attemptedUpload, setAttemptedUpload] = useState(false)
   const fileInputRef = useRef<HTMLInputElement>(null)
   const dispatch = useDispatch<AppDispatch>()
-  const productId = useSelector((state: ProductState) => state.productId)
-  const formErrors = useSelector((state: ProductState) => state.formErrors)
+  const productId = useSelector(
+    (state: ProductState) => state.newProduct.productId
+  )
+  const formErrors = useSelector(
+    (state: ProductState) => state.newProduct.formErrors
```
Comment (pqoqubbw) [inline src/components/Crm-images/index.tsx:33]: same question

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx (hunk 2 above)
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx]: why it is `ProductState` and not `RootState`?

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx (hunk 1 above)
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx]: you forgot type for `productId`

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx (hunk 3 above)
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx]:
```suggestion
           console.log(error)
```

File: src/components/Crm-add-new-product-table/CrmAddNewProductTable.tsx [hunk 5]
```diff
@@ -40,10 +41,10 @@ const CrmAddNewProductTable = () => {
   ])
   const dispatch = useDispatch<AppDispatch>()
   const productId = useSelector(
-    (state: RootState) => state.newProduct.productId
+    (state: ProductState) => state.newProduct.productId
   )
   const validationErrors = useSelector(
-    (state: RootState) => state.newProduct.formErrors
+    (state: ProductState) => state.newProduct.formErrors
```
Comment (pqoqubbw) [inline src/components/Crm-add-new-product-table/CrmAddNewProductTable.tsx:47]: same question

File: src/components/Crm-images/index.tsx (hunk 4 above)
Comment (pqoqubbw) [inline src/components/Crm-images/index.tsx]: same question

**Review by pqoqubbw — CHANGES_REQUESTED** [review]

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx (hunk 1 above)
Comment (Tkachenko01001) [reply src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx]: done

File: src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx (hunk 3 above)
Comment (Tkachenko01001) [reply src/components/Crm-add-new-product-button/CrmAddNewProductButton.tsx]: done

File: src/components/Crm-add-new-product-table/CrmAddNewProductTable.tsx (hunk 5 above)
Comment (Tkachenko01001) [reply src/components/Crm-add-new-product-table/CrmAddNewProductTable.tsx:47]: done

File: src/components/Crm-images/index.tsx (hunk 4 above)
Comment (Tkachenko01001) [reply src/components/Crm-images/index.tsx]: done

**Pull request merged**